examples/standalone/acoustic_comms_demo/README.md

```diff
@@ -15,17 +15,19 @@ will receive the signal and start moving before Daphne.
 
 From this directory, run the following to compile:
 
-    mkdir build
-    cd build
-    cmake ..
-    make
+```bash
+cmake -B build
+cmake --build build
+```
 
 ## Execute Instructions
 
-From the `build` directory, run Gazebo Sim and the example controller:
+Run Gazebo Sim and the example controller:
 
-    gz sim -r ../../../worlds/acoustic_comms_demo.sdf
-    ./acoustic_comms_demo
+```bash
+gz sim -r ../../worlds/acoustic_comms_demo.sdf
+./build/acoustic_comms_demo
```

Comment:
I haven't tried this, but sometimes, for these examples, where you run the application matters because included files such as plugins or mesh files are specified relative to the CWD. I'm okay with making this change, just wanted to ask if you've verified that this works.

Reply:
Yep. I went through and tested everything. Some of them didn't work. All the ones I changed should be working.

Reply:
That's great. Thanks!